modules/presets/index.js

```diff
@@ -72,6 +72,24 @@ export function presetIndex() {
           presets: vals[2],
           fields: vals[3]
         });
+
+        // Add OpenHistoricalMap-specific fields.
```

Comment:
We should probably find a better place for this. Eventually there could be a lot of overrides here, maybe not enough to require loading a separate JSON file from a CDN, but enough to get awkward buried in this function.